contracts/OlasMech.sol

```diff
@@ -338,6 +316,7 @@ abstract contract OlasMech is Mech, IErrorsMech, ImmutableStorage {
         return multisig == getOperator();
     }
 
+    // TODO remove?
```

Comment:
This function seems to be overly complicated, but we can leave it not to perform those calculations via the script, if needed.

Reply:
yes we should offer simple getters for info off-chain mech needs